ht-8/index.js

```diff
@@ -0,0 +1,31 @@
+function initCollapse(){
+  const buttons = Array.from(document.querySelectorAll(`[data-toggle="collapse"]`));
+  buttons.forEach(button => {
+    button.addEventListener('click', () => {
+      const cardId = button.getAttribute('href') || button.dataset.target;
+      collapseShow(cardId, button);
+    })
+  });
+  const collapseShow = (cardId, button) => {
+    const parentButton = button.parentElement.parentElement.parentElement;
+    const cards = Array.from(parentButton.querySelectorAll(cardId));
+    const cardShow = parentButton.querySelector('.show');
+    cards.forEach(card => {
```

Comment:
let's try to avoid using forEach. try to find a way update only necessary elements